content/docs/deploy/core/upgrading.mdx

```diff
@@ -13,6 +13,48 @@ sidebar_position: 10
 
 # Upgrade guide
 
+## 0.24.0
+
+### Breaking
+
+#### Set Authorization Header
+
+The deprecated `set_authorization_header` configuration setting has been removed. You can use the [Set Request Headers](/docs/reference/routes/headers#set-request-headers) setting to pass IdP tokens to upstream services in any header.
+
+#### Base64-encoded Certificates
+
+Previously, the `certificates` key supported base64-encoded certificates as a value (this option was not documented). We've removed support for base64-encoded certificates for this setting and now require that you only use the relative file location.
+
+**Note:** The `certificates` key is distinct from the `certificate` key. The `certificate` key setting still supports base64-encoded certificates; the `certificates` _list_ does not.
+
+See [Certificates](/docs/reference/certificates#certificates) for more information.
+
+#### Redis Storage Backend
+
+PostgreSQL has been the recommended databroker storage backend since v0.18. Support for Redis has now been removed.
+
+See [Persistence](/docs/internals/data-storage#postgres) for more information.
+
+### Performance
+
+v0.24.0 includes several performance enhancements for Pomerium Core. See the sections below for more information.
+
+#### Databroker state sync
```

Comment:
I think this is also a console-specific change? @wasaga, can you confirm?

Reply:
Yeah I wasn't sure about this one. I can remove if it is just Console.